src/server/api/routers/webhook.ts

```diff
@@ -1,13 +1,15 @@
 import { Octokit } from "@octokit/rest";
 import { z } from "zod";
+import { env } from "~/env.mjs";
 import { createTRPCRouter, protectedProcedure } from "~/server/api/trpc";
 
 export const webhookRouter = createTRPCRouter({
   createWebhook: protectedProcedure.input(z.object({ accessToken: z.string()})).mutation(async ({ input }) => {
     const owner = "GabrielPedroza";
-    const repo = "e-commerce";
+    const repo = "exotica";
 
-    const WEBHOOK_URL = "https://bread-meta.vercel.app/api/webhooks/webhook";
+    const WEBHOOK_URL = "https://hkdk.events/G3KE7hkFpr5I";
```

Comment:
what is hkdk.events?

Reply:
This is the webhook proxy. Refer to https://github.com/GabrielPedroza/bread/pull/9